deb/common/docker-ce.maintscript

```diff
@@ -0,0 +1,3 @@
+# *technically* this "5:24~" should be something more specific like "5:24.0.5-1~" (the first released version to introduce this "rm_conffile" line), but as long as it's strictly *newer* than the last version to ship the conffile and stricly *older* than the first version with the rm_conffile line, it should work fine (so we're using the will-never-exist non-semver version of "24" as our in-between release for the 23.x.x to 24.x.x)
+rm_conffile /etc/init/docker.conf 5:24~
```

Comment:
This should actually be `5:25~` for 25+ and a much more specific minor version for 24.x backport, right?

Reply:
I was intending to apply this commit to 24.0 as we haven't branched yet; it sounds like we should go with the patch version here, and only use the major version in the 25.0 branch?

Reply:
Yeah, fair - so this should either be `5:24.0.5~` (which would honestly be fine for both 24.x backport and 25.x+ if you want the same value in both and we actually get this into 24.0.5) or you could `5:25~` for 25.x+.

Reply:
(The problem is that as-is, a bare "24" is _not_ newer than 24.0.5, so doesn't satisfy my long comments :smile:)

Reply:
Check the last push, it's currently `rm_conffile /etc/init/docker.conf 5:24.0.5-1~`